examples/with-afterjs/src/server.js

```diff
@@ -0,0 +1,34 @@
+import polka from 'polka';
+import { render } from '@jaredpalmer/after';
+import serve from 'serve-static';
+import routes from './routes';
+
+const assets = require(process.env.RAZZLE_ASSETS_MANIFEST);
+const statics = serve(process.env.RAZZLE_PUBLIC_DIR);
+
+const server = polka();
+
+server
+  .use(statics)
+  .get('/*', async (req, res) => {
+    try {
+      const html = await render({
+        req,
+        res,
+        routes,
+        assets,
+        // Anything else you add here will be made available
+        // within getInitialProps(ctx)
+        // e.g a redux store...
+        customThing: 'thing',
+      });
+      res.end(html);
+    } catch (error) {
+    	let json = JSON.stringify(error);
```

Comment:
Worth mentioning that this "uglier" bit could easily be replaced with [`@polka/send-type`](https://github.com/lukeed/polka/tree/master/packages/send-type), but I didn't want to hide what was going on.